src/events/pull/index.ts

```diff
@@ -142,17 +145,16 @@ async function createOrUpdateComment(
   });
 
   const botComment = comments.find((c) => {
-    return c.user.login === commenter;
+    return c.user?.login === commenter;
   });
 
   if (botComment === undefined) {
     await context.octokit.issues.createComment(context.issue({ body }));
   } else {
     // Update.
-    botComment.body = body;
     const comment = {
       ...context.issue(),
```

Comment:
In fact, there is no need for `issue_number` here, pass in `context.repo()` is ok.

Reply:
Fixed.